src/workloads/sharding/WriteOneWithoutShardKeyShardedCollection.yml

```diff
@@ -0,0 +1,127 @@
+  BalancerWait: &BalancerWait "20 seconds"  # Wait out the 20 second delay between balancing rounds
+  # with a little extra leeway for balancing to finish.
+
+Actors:
+- Name: CreateShardedCluster
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationMetricsName: EnableSharding
+      OperationName: AdminCommand
+      OperationCommand:
+        enableSharding: *Database
+    - OperationMetricsName: ShardCollection
+      OperationName: AdminCommand
+      OperationCommand:
+        shardCollection: *Namespace
+        key: {x: 1}
+    - OperationMetricsName: BalancerStart
+      OperationName: AdminCommand
+      OperationCommand:
+        balancerStart: 1
+  - *Nop
+#  - *Nop
+#  - *Nop
+#  - *Nop
+
+- Name: LoadInitialDataOnShardedCollection
+  Type: MonotonicSingleLoader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    SleepBefore: *BalancerWait
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    Database: *Database
+    Document: {x: 0, y: 0, z: 0}
+  # - *Nop
+  # - *Nop
+  # - *Nop
+
+# - Name: CRUDOpsOnShardedCollection
+#   Type: CrudActor
+#   Threads: 1  # We want to use 1 thread to avoid updates throwing WriteConflict errors.
+#   Phases:
+#   - *Nop
+#   - *Nop
+#   - Repeat: *DocumentCount
+#     Database: *Database
+#     Collection: *ShardedCollection
+#     Operations:
+#     - OperationMetricsName: UpdateOneWithoutShardKeyShardedCollection
+#       OperationName: updateOne
+#       OperationCommand:
+#         Filter: {y: 0}
+#         Update: {$inc: {y: 1}}
+#         Options:
+#           WriteConcern:
+#             Level: majority
+#             TimeoutMillis: 5000
+#       OutcomeCounts:
+#       - Filter: {y: 1}
+#         Count: 1000
+#   - Repeat: *DocumentCount
+#     Database: *Database
+#     Collection: *ShardedCollection
+#     Operations:
+#     - OperationMetricsName: FindAndModifyOneWithoutShardKeyShardedCollection
+#       OperationName: findOneAndUpdate
+#       OperationCommand:
+#         Filter: {y: 1}
+#         Update: {$inc: {y: 1}}
+#         Options:
+#           WriteConcern:
+#             Level: majority
+#             TimeoutMillis: 5000
+#         OutcomeCounts:
+#         - Filter: {y: 2}
+#           Count: 1000
+#   - Repeat: *DocumentCount
+#     Database: *Database
+#     Collection: *ShardedCollection
+#     Operations:
+#     - OperationMetricsName: DeleteWithoutShardKeyShardedCollection
+#       OperationName: deleteOne
+#       OperationCommand:
+#         Filter: {z: 0}
+#         Options:
+#           WriteConcern:
+#             Level: majority
+#             TimeoutMillis: 5000
+#       OutcomeCounts:
+#       - Filter: {z: 0}
+#         Count: 0
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - shard
+      - shard-lite
+      - shard-lite-all-feature-flags
+    branch_name:
+      $neq:
+      - v4.0
+      - v4.2
+      - v4.4
+      - v5.0
+#      - v6.0
```

Comment:
Is this going to stay commented?

Reply:
I'm not 100 percent sure if it tests on non major releases. This feature won't be on 6.0 it's only available on master.